### DISCUSSION TITLE
Prisma error with suggestion in VS code

### DISCUSSION THREAD

**Discussion by IRediTOTO** [prisma/prisma#13440] [Q&A]
Hi, I am getting error with typescript. In VS code, there is no suggestion for prisma type in node_modules
Types was generated but not can't import it. I can't undestand why.  Some types outside `namespace Prisma` like Category type can import normal. 
Here is my tsconfig.json, I am using node `v14.19.2`
```
{
  "compilerOptions": {
    "target": "es5",
    "lib": ["dom", "dom.iterable", "esnext"],
    "allowJs": true,
    "skipLibCheck": true,
    "strict": false,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve"
  },
  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
  "exclude": ["node_modules"]
}

```

![image](https://user-images.githubusercontent.com/15695252/169675909-c480400f-b859-4f50-8f13-92d6b0072b84.png)

**Comment by IRediTOTO** [comment]
Oh my bad, it should be `Prisma.CategoryWhereInput`

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
